modules/apps/feature-flag/feature-flag-web/src/FeatureFlags.testcase

```diff
@@ -0,0 +1,79 @@
+@component-name = "feature flags"
+definition {
+
+	property portal.acceptance = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Feature Flags";
+
+	setUp {
+		User.firstLoginPG();
+
+		task ("Given the Feature Flags page") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "Instance Settings");
+
+			Click(
+				key_itemName = "Feature Flags",
+				locator1 = "ListGroupItem#ITEM_TEXT");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-186037 - This tests asserts that a feature flag can be Disabled"
+	@priority = 5
+	test CanBeDisabled {
+		property portal.acceptance = "true";
+
+		task ("And a feature flag is enabled") {
+			Click(
+				key_labelName = "Disabled",
+				locator1 = "CalendarLookAndFeelConfiguration#TOGGLE_SWITCH_LABEL");
+		}
+
+		task ("When the administrator disables a feature flag") {
+			Click(
+				key_labelName = "Enabled",
+				locator1 = "CalendarLookAndFeelConfiguration#TOGGLE_SWITCH_LABEL");
+		}
+
+		task ("Then assert the toggle is disabled state") {
+			AssertElementPresent(
+				key_labelName = "Disabled",
+				locator1 = "CalendarLookAndFeelConfiguration#TOGGLE_SWITCH_LABEL");
+		}
+	}
+
+	@description = "LPS-186037 - This tests asserts that a feature flag can be enabled"
+	@priority = 5
+	test CanBeEnabled {
+		property portal.acceptance = "true";
+
+		task ("And a feature flag is disabled") {
+			AssertElementPresent(
```

Comment:
I suggest creating an if else statement to toggle to disabled if the feature flag is enabled. 

And then use VerifyElementPresent on a path something like this:

`//*[contains(text(),'${title-row-name}')]//ancestor::dd//label//span[contains(@class,'toggle-switch-label')]`